proposals/0251-font-styles.md

```diff
@@ -7,92 +7,73 @@
 
 ## Introduction
 
-This feature would let apps bold, underline or italicize text within Alert, Show or Perform Interaction RPCS.
+This feature would let apps style text within `Alert`, `Show`, `ScrollableMessage` or `PerformInteraction` RPC with things like bold, italics or underlining.
 
 ## Motivation
 
 As an app partner, I want to be able to customize text on main layouts and popups so that I can have a visually pleasing and easy to use UI.
 
 ## Proposed solution
 
-
-When an app wants a more visually appealing UI experience,  it can add a font style customization onto each line of text if desired.  If no font style is provided, then it is assumed that the font is normal.  
+When an app wants a more visually appealing UI experience,  it can add a HTML tags for font style customization onto text if desired.
 
 ![Example Screen](../assets/proposals/0251-font-styles/0251-font-styles.png)
 
-Add a new struct `TextStyle`
+### Hyper Text Capabilities
+
+The HMI should be able to provide a list of supported hyper text elements to SDL and to  the applications. This allows the HMI to have it's own flexibility to support what it wants:
+
+### HMI & Mobile API changes
 
 ```xml
-<struct name="TextStyle">
-  <param name="bold" type="Boolean" mandatory="false">
-	<description>Bold the line of text. Defaults to false.</description>	
-  </param>
-  <param name="italic" type="Boolean" mandatory="false">
-	<description>Italicize the line of text. Defaults to false.</description>
-  </param>
-  <param name="underline" type="Boolean" mandatory="false">
-	 <description>Underline the line of text. Defaults to false.</description>
-  </param>
-</struct>	
+<struct name="TextField" since="1.0">
+:
+    <param name="hyperTextSupported" type="String" array="true" mandatory="false" minvalue="1" maxvalue="100" since="6.x">    
+    </param>
+</struct>
 ```
 
-Add to the TextFieldStruct - this will cover Show, Alert and TBT RPCs, as well as `InitialText` for various RPCs.
+The HMI API equivalent should exclude the versioning attributes.
 
+Alternative: An enum can be added that lists all the hyper text elements.
+
+With this implementation, technically the html tags take up part of the 500 character count restriction placed in `TextFieldStruct`. So, the character count for `TextFieldStruct` would go from 500 to 1000 in the HMI API. This struct is used for pretty much any text field in RPCs like `Alert`, `ScrollableMessage`, `Show` and so on.
```

Comment:
Please make sure you're explicitly stating each MOBILE_API change, as requested in item 3 [here](https://github.com/smartdevicelink/sdl_evolution/issues/828#issuecomment-538036957).

Reply:
Doing this for TextFieldStruct works for everything we need